DuoUniversal.Tests/TestCertPinning.cs

```diff
@@ -76,7 +76,7 @@ public void Setup()
         [Test]
         public void TestReadCertFile()
         {
-            Assert.AreEqual(10, CertificatePinnerFactory.ReadCertsFromFile().Length);
+            Assert.AreEqual(1, CertificatePinnerFactory.ReadCertsFromFile().Length);
```

Comment:
I count 15 certificates in the ca_certs.pem file, so that should be the expected value.  See my comments on that file